doc/nrf/ug_gzp.rst

```diff
@@ -7,8 +7,16 @@ Gazell Pairing
    :local:
    :depth: 2
 
-The Gazell Pairing library for the nRF5 Series enables applications to use the Gazell Link Layer to provide a secure wireless link between Gazell nodes.
-The library is customized for pairing a device (for example, a mouse, keyboard, or remote control) with a host (typically a USB dongle) using Gazell.
+The Gazell Pairing library for the nRF52 Series enables applications to use the Gazell Link Layer to provide a secure wireless link between Gazell nodes.
+The library is customized for pairing a Device (for example, a mouse, keyboard, or remote control) with a Host (typically a USB dongle) using Gazell.
+
+Additionally, Gazell Pairing uses additional hardware peripherals and restricts the usage of the Gazell Link Layer.
```

Comment:
Do we need to start with Additionally, as additional stuff follows?

Reply:
Perhaps not. I agree that does sound repetitive. I will rewrite so that it reads,
`Gazell Pairing also uses additional hardware...`, removing "Additionally".